db/seeds.rb

```diff
@@ -0,0 +1,51 @@
+require "faker"
+
+lambeth = LocalAuthority.find_or_create_by!(
+  name: "Lambeth Council",
+  subdomain: "lambeth",
+  signatory_name: "Christina Thompson",
+  signatory_job_title: "Director of Finance & Property",
+  enquiries_paragraph: "Postal address: Planning London Borough of Lambeth PO Box 734 Winchester SO23 5DG.",
+  email_address: "planning@lambeth.gov.uk",
+)
+southwark = LocalAuthority.find_or_create_by!(
+  name: "Southwark Council",
+  subdomain: "southwark",
+  signatory_name: "Simon Bevan",
+  signatory_job_title: "Director of Planning",
+  enquiries_paragraph: "Postal address: Planning London Borough of Southwark PO Box 734 Winchester SO23 5DG.",
+  email_address: "planning@lambeth.gov.uk",
+)
+
+bucks = LocalAuthority.find_or_create_by!(
+  name: "Buckinghamshire",
+  subdomain: "bucks",
+  signatory_name: "Steve Bambick",
+  signatory_job_title: "Director of Planning",
+  enquiries_paragraph: "Postal address: Planning Buckinghamshire Council, Gatehouse Rd, Aylesbury HP19 8FF",
+  email_address: "planning@buckinghamshire.gov.uk",
+)
+
+ApiUser.find_or_create_by!(name: "api_user", token: (ENV["API_TOKEN"] || "123"))
+
+admin_roles = %i[assessor reviewer]
+local_authorities = [southwark, lambeth, bucks]
+
+local_authorities.each do |authority|
+  admin_roles.each do |admin_role|
+    User.find_or_create_by!(email: "#{authority.subdomain}_#{admin_role}@example.com") do |user|
+      first_name = Faker::Name.unique.first_name
+      last_name = Faker::Name.unique.last_name
+      user.name = "#{first_name} #{last_name}"
+      user.local_authority = authority
+      if Rails.env.development?
+        user.password = user.password_confirmation = "password"
+      else
+        user.password = user.password_confirmation = SecureRandom.uuid
+        user.encrypted_password =
+          "$2a$11$uvtPXUB2CmO8WEYm7ajHf.XhZtBsclT/sT45ijLMIELShaZvceW5."
+      end
+      user.role = admin_role
+    end
+  end
+end
```

Comment:
I just pulled the branch and reset the db - now there are no planning applications, which we were creating before with create_sample_data?
Also I remember there used to be a dependency on create_sample_data during the build process, I guess that's been fixed now?

Reply:
I explained my reasoning in the comment to https://github.com/unboxed/bops/pull/205/commits/f0d5556c2ea695b38e02adc1a043f57ff7eb3e8a

create_sample_data was never in the build process

Reply:
I get we are not using it for build but I still don't have any planning applications to test locally?